terms/structuralriskminimization_maf.tex

```diff
@@ -0,0 +1,99 @@
+\chapter{\textit{}Structural Risk Minimization}
+\label{ch:structuralriskminimization}\index{structuralriskminimization}
+
+
+\section Overview \newline \index{overview}
```

Comment:
\section requires curly brackets -- i.e. `\section{something in brackets}`
do not use newline.  And it doesnt make sense to have "overview" in the index entry ... overview of what ???